tests/hpc/slurm_slave.pm

```diff
@@ -30,6 +30,8 @@ sub run {
 
     # Install slurm
     zypper_call('in slurm-munge');
+    # try to install slurm-node, even if not available
+    type_string("zypper -n in slurm-node\n");
```

Comment:
I would vote for soft-failure which checking if package is actually installed

Reply:
and what you want to link to that?
according to egbert, this is not wrong if the package is not there